R/cleaning.R

```diff
@@ -279,56 +312,97 @@ validate_postcode <- function(postcode) {
 #'
 #' Replaces missing and codes that are not valid with the invalid code value.
 #'
-#' @param field_in Variable to be validated.
+#' @param x Variable to be validated.
 #' @param valid_codes List of valid codes.
 #' @param invalid_code Invalid code to replace values not observed in valid_codes with.
 #' @returns Vector of codes where values not in valid_codes are replaced with invalid_code.
 #' @export
-validate_codes <- function(field_in, valid_codes, invalid_code) {
-  return(replace(field_in, (!is.na(field_in) & !(field_in %in% valid_codes)), invalid_code))
+#' @examples
+#' x <- c("A001", "B002", "C003", "A001", "D002", "A002")
+#' valid_codes <- c("A001", "A002", "B001", "B002", "C003")
+#' invalid_code <- "M999"
+#' validate_codes(x, valid_codes, invalid_code)
+validate_codes <- function(x, valid_codes, invalid_code) {
+  return(replace(x, (!is.na(x) & !(x %in% valid_codes)), invalid_code))
 }
 
 #' Tidies dates
 #'
 #' Replaces "1900-01-01" with NA and anything < "1888-01-01" or > "2023-03-31" as NA.
 #'
-#' @param field_in Date variable to tidy.
-#' @param start_date The first date from which values are valid, anything before this is converted to NA.
-#' @param finish_date The last date from which values are valid, anything after this is converted to NA.
+#' @param x Date variable to tidy.
+#' @param start_date The first date from which values are valid, anything before this is converted to NA. Default is
+#' "1888-01-01".
+#' @param finish_date The last date from which values are valid, anything after this is converted to NA. Default is
+#' "2017-03-31".
```

Comment:
update default to 2023-03-31
```suggestion
#' "2023-03-31".
```